rpm/cray/csm/sle-15sp5/index.yaml

```diff
@@ -23,10 +23,11 @@
 #
 https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/sle-15sp5/:
   rpms:
-    - bos-reporter-2.5.5-1.noarch
     - cf-ca-cert-config-framework-2.6.0-1.noarch
     - cfs-debugger-1.4.0-1.noarch
     - cfs-state-reporter-1.9.3-1.noarch
     - cfs-trust-1.6.7-1.noarch
     - csm-ssh-keys-1.5.5-1.noarch
     - csm-ssh-keys-roles-1.5.5-1.noarch
+    - hpe-yq-4.33.3-1.aarch64
```

Comment:
Seeing this add/remove drew my attention to the fact that this also needs to cut-over, PR is going up for that now.

Reply:
If it gets ready in time, I can include it in these PRs. Otherwise, we can do it in another PR.

Reply:
I see that the `noos` versions of the RPM are published, so I've updated the two `csm` PRs accordingly.